runtime/doc/lua.txt

```diff
@@ -573,10 +573,180 @@ VIM.LPEG                                                            *lua-lpeg*
+
+    Parameters: ~
+      • {v} (string|integer) The rule index or name.
+
+vim.lpeg.locale(tab)                                          *vim.lpeg.locale()*
+    Returns a table with patterns for matching character classes according to the current locale.
+
+    Parameters: ~
+      • {tab} (table|nil) Optional table to create fields in.
+
+vim.lpeg.C(patt)                                                   *vim.lpeg.C()*
+    Creates a simple capture, which captures the substring of the subject that matches `patt`.
+
+    Parameters: ~
+      • {patt} (vim.lpeg.Pattern) The pattern to capture.
+
+vim.lpeg.Carg(n)                                                   *vim.lpeg.Carg()*
+    Creates an argument capture.
+
+    Parameters: ~
+      • {n} (integer) The index of the extra argument.
+
+vim.lpeg.Cb(name)                                                 *vim.lpeg.Cb()*
+    Creates a back capture.
+
+    Parameters: ~
+      • {name} (any) The name of the capture.
+
+vim.lpeg.Cc(...)                                                   *vim.lpeg.Cc()*
+    Creates a constant capture.
+
+    Parameters: ~
+      • {...} (any) Values to capture.
+
+vim.lpeg.Cf(patt, func)                                       *vim.lpeg.Cf()*
+    Creates a fold capture.
+
+    Parameters: ~
+      • {patt} (vim.lpeg.Pattern) The pattern to fold.
+      • {func} (function) The fold function.
+
+vim.lpeg.Cg(patt, name)                                     *vim.lpeg.Cg()*
+    Creates a group capture.
+
+    Parameters: ~
+      • {patt} (vim.lpeg.Pattern) The pattern to group.
+      • {name} (string|nil) The name of the group.
+
+vim.lpeg.Cp()                                                          *vim.lpeg.Cp()*
+    Creates a position capture.
+
+vim.lpeg.Cs(patt)                                                  *vim.lpeg.Cs()*
+    Creates a substitution capture.
+
+    Parameters: ~
+      • {patt} (vim.lpeg.Pattern) The pattern to capture.
+
+vim.lpeg.Ct(patt)                                                  *vim.lpeg.Ct()*
+    Creates a table capture.
+
+    Parameters: ~
+      • {patt} (vim.lpeg.Pattern|'') The pattern to capture.
+
+vim.lpeg.Cmt(patt, fn)                                         *vim.lpeg.Cmt()*
+    Creates a match-time capture.
+
+    Parameters: ~
+      • {patt} (vim.lpeg.Pattern) The pattern to capture.
+      • {fn} (function) The match-time function.
+
+vim.lpeg.version()                                                *vim.lpeg.version()*
+    Returns a string with the running version of LPeg.
+
+vim.lpeg.setmaxstack(max)                                 *vim.lpeg.setmaxstack()*
+    Sets a limit for the size of the backtrack stack used by LPeg.
+
+    Parameters: ~
+      • {max} (integer) The maximum size of the stack.
+
+------------------------------------------------------------------------------
+VIM.RE                                                              *lua-re*
```

Comment:
```suggestion
VIM.RE                                                              *vim.re*
```
(and squash the commits; commit linter requires a space after the colon: `docs(lua): add docs for vim.lpeg, vim.re`)

Reply:
okay let me fix that